config/routes.rb

```diff
@@ -24,6 +24,7 @@
     resources :orders, only: [:index, :show, :edit]
     resources :items, only: [:index, :show, :edit, :new, :create, :update]
     resources :category_items, only: [:new, :create]
+    get '/category_items/new_category_to_item/:item_id', to: 'category_items#new_category_to_item', :as => 'new_category_to_item'
```

Comment:
Line is too long. [129/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.